projects/Mallard/src/components/article/types/article.tsx

```diff
@@ -28,8 +29,12 @@ const styles = StyleSheet.create({
         /*
         The below line fixes crashes on Android
```

Comment:
wait, but then it's going to crash on Android 9, no?

Reply:
Look at the lines below. It checks for Android 9 and applies the `0.99` fix.